src/Pandell.PackageRunner/Program.cs

```diff
@@ -263,5 +266,12 @@ private static Parameters ParseArguments(IEnumerable<string> args)
 
             return parameters;
         }
+
+        /// <summary>
+        /// </summary>
+        private static byte[] HexStringToByteArray(string hex)
+        {
+            return Enumerable.Range(0, hex.Length/2).Select(x => Convert.ToByte(hex.Substring(x*2, 2), 16)).ToArray();
```

Comment:
Actually, because we control the format of the token, there are two options:

1. Generate byte array in `Token.cs` directly. This works well with my MSBuild C# task recommendation above

2. Or, use Base64 encoding for the token

In either case, you can delete this method.